stylesheets/components/badge.css

```diff
@@ -0,0 +1,38 @@
+.badge {
+  background-color: var(--contrast-color);
+  color: black;
+  height: 1.25rem;
+  width: 1.25rem;
+  line-height: 1.25rem;
+  border-radius: 50%;
+  text-align: center;
+  position: absolute;
+  top: -0.75rem;
+  right: -0.75rem;
+  font-size: small;
+  font-weight: 600;
+}
+
+.badge-avatar {
+  top: -2rem;
+  right: 0;
+}
+
+.badge-large {
+  width: 2rem;
+  line-height: 2rem;
+  height: 2rem;
+  font-size: 1.125rem;
+}
+
+.badge-left {
+  left: -0.75rem;
+}
+
+.position-absolute {
```

Comment:
This generalized classes can be moved to base class or main class, as these can be used for other components.